bindings/src/main/scala/chrome/pageAction/PageAction.scala

```diff
@@ -13,10 +13,10 @@ object PageAction {
 
   val onClicked: EventSource[Tab] = bindings.PageAction.onClicked
 
-  def show(tabId: Tab.Id): Unit =
+  def show(tabId: js.UndefOr[Int] = js.undefined): Unit =
     bindings.PageAction.show(tabId)
 
-  def hide(tabId: Tab.Id): Unit =
+  def hide(tabId: js.UndefOr[Int] = js.undefined): Unit =
```

Comment:
Also, while I the `UndefOr[_]` seems reasonable for the bindings layer, because it's the plain js facade, here I would add two methods and avoid exposing js types, the method with and without the argument.

The layers above bindings are supposed to be Scala idiomatic so that devs don't need to deal with the js complextities.